test/cppsuite/tests/schema_sweep.cpp

```diff
@@ -0,0 +1,147 @@
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ * OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "src/common/constants.h"
+#include "src/common/logger.h"
+#include "src/main/test.h"
+
+namespace test_harness {
+
+// The intent of the test is to stress the code paths related to schema operations and dhandles
+// management. This test keeps performing schema operations while reconfiguring the sweep server.
+class schema_sweep : public test {
+public:
+    schema_sweep(const test_args &args) : test(args)
+    {
+        init_operation_tracker();
+    }
+
+    // Reconfigures the connection with different sweep server parameters.
+    void
+    custom_operation(thread_worker *tw) override final
+    {
+        WT_CONNECTION *conn = connection_manager::instance().get_connection();
+        bool aggressive_sweep = false;
+        const std::string aggressive_sweep_cfg =
+          "file_manager=(close_handle_minimum=0,close_idle_time=1,close_scan_interval=1)";
+        const std::string default_sweep_cfg =
+          "file_manager=(close_handle_minimum=250,close_idle_time=30,close_scan_interval=10)";
+
+        while (tw->running()) {
+            tw->sleep();
+            testutil_check(conn->reconfigure(
+              conn, aggressive_sweep ? default_sweep_cfg.c_str() : aggressive_sweep_cfg.c_str()));
+            aggressive_sweep = !aggressive_sweep;
+        }
+    }
+
+    // Keeps creating collections.
+    void
+    insert_operation(thread_worker *tw) override final
+    {
+        auto max_coll_count = 1000;
+        while (tw->running()) {
+            if (tw->db.get_collection_count() < max_coll_count) {
+                auto session = connection_manager::instance().create_session();
+                tw->db.add_collection(session);
+            }
+            tw->sleep();
+        }
+    }
+
+    // Keeps deleting collections.
+    void
+    remove_operation(thread_worker *tw) override final
+    {
+        while (tw->running()) {
+            if (tw->db.get_collection_count() != 0) {
+                std::string cfg = "force=";
+                cfg += random_generator::instance().generate_bool() ? "true" : "false";
+                // We don't need to check whether the collection has been actually removed or not.
+                tw->db.remove_random_collection(cfg);
+            }
+            tw->sleep();
+        }
+    }
+
+    // Selects a random collection and performs an update on it. Note that the collection can be
+    // deleted while the update is happening.
+    void
+    update_operation(thread_worker *tw) override final
+    {
+        while (tw->running()) {
+            tw->sleep();
+            {
```

Comment:
Each iteration of this while loop is considered it's own scope which means we'll auto clean up all sessions/cursors at the end of each loop iteration automatically (thanks @jeremythorp for confirming!). 
We can drop these braces here and get the same auto-cleanup behaviour

Reply:
Thank you! Updated in f9e56e842310e575658cb9bf7cbd2135f7a79db5